README.md

```diff
@@ -10,7 +10,7 @@ when using the **--inplace** option.
 ### Installing with DUB
 
 ```sh
-> dub fetch --version='~master' dfmt && dub run dfmt -- -h
+> dub fetch dfmt@"~master" && dub run dfmt -- -h
```

Comment:
Since a couple of versions, Dub will automatically fetch a package, if it doesn't exist, so we can skip the `fetch` part. Also I believe the `~master` suggestion is outdated - some time ago there was a big gap between the last official release and `master`, but I think it's not the case anymore.

```suggestion
> dub run dfmt -- -h
```

Reply:
I committed the suggestion, it's a more straightforward way to install and run dfmt.